apis/datacatalog/v1alpha1/entry_types.go

```diff
@@ -325,6 +307,24 @@ type DataCatalogEntryObservedState struct {
 	// +kcc:proto:field=google.cloud.datacatalog.v1.Entry.name
 	// Name *string `json:"name,omitempty"`
 
+	// The resource this metadata entry refers to.
+	//
+	//  For Google Cloud Platform resources, `linked_resource` is the
+	//  [Full Resource Name]
+	//  (https://cloud.google.com/apis/design/resource_names#full_resource_name).
+	//
+	//  Output only when the entry is one of the types in the `EntryType` enum.
+	//
+	//  For entries with a `user_specified_type`, this field is optional and
+	//  defaults to an empty string.
+	//
+	//  The resource string must contain only letters (a-z, A-Z), numbers (0-9),
+	//  underscores (_), periods (.), colons (:), slashes (/), dashes (-),
+	//  and hashes (#).
+	//  The maximum size is 200 bytes when encoded in UTF-8.
+	// +kcc:proto:field=google.cloud.datacatalog.v1.Entry.linked_resource
+	LinkedResource *string `json:"linkedResource,omitempty"`
```

Comment:
No action needed.

I'm good with making this field output-only to begin with, though from the field description, it looks like this field can be user-specified in some use cases. I think we'll want to support it under spec eventually.